src/components/expand-viewer/index.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FC, useState } from 'react';
+import Popup, { PopupProps } from '../popup';
+import IconButton from '../icon-button';
+import { getIcon, IconName } from '../icon/list';
+import { useTheme } from 'emotion-theming';
+import { Box, Flex } from 'rebass';
+import Value from '../typography/value';
+import Subtitle from '../typography/subtitle';
+import { expandViewButtonStyle } from './expand-viewer.styles';
+import { ITheme } from '../../theme/types';
+
+// ExpandViewer
```

Comment:
nit: do we need this comment? It doesn't seem to serve any value, as the file name and the component name speak for themselves

Reply:
yeah forgot to remove it, it's here since first I developed this component inside the Code component then later move it out to its own module.